test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -12214,7 +12214,20 @@ def forward(self, x):
         self.run_test(Module(True), x, rtol=1e-3, atol=1e-6)
 
     @skipIfUnsupportedMinOpsetVersion(16)
-    def test_grid_sample(self):
+    @common_utils.parametrize(
+        "mode",
+        ("bilinear", "nearest", "bicubic"),
+        name_fn=lambda mode: mode,
```

Comment:
According to the doc, we don't need the name_fn for simple cases: https://sourcegraph.com/github.com/pytorch/pytorch@bcc02769bef1d7b89bec724223284958b7c5b564/-/blob/torch/testing/_internal/common_utils.py?L248:7&subtree=true

Reply:
You are right. Only leaving **align_corners** as it would be ambiguous without naming.